src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,83 @@
+import MenuButton from 'components/Buttons/MenuButton';
+import { menu } from 'data/menu';
+import { Component } from 'react';
+import styled from 'styled-components';
+
+export default class Navbar extends Component {
+  render() {
+    return (
+      <Header>
+        <HeaderContainer>
+          <form>
+            <FormGroup>
+              <img
+                className='searchIcon'
+                src='/images/search.svg'
```

Comment:
import image and add it to src
why are you using className? you should be using styled-components